kmod/patch/livepatch-patch-hook.c

```diff
@@ -74,6 +74,11 @@
 # define HAVE_SIMPLE_ENABLE
 #endif
 
+/* TODO: check RHEL_RELEASE_CODE for KLP_REPLACE*/
```

Comment:
Atomic replace was backported to rhel-8's kernel-4.18.0-147.3.el8.  It was never backported to rhel-7.

Reply:
I think we can do `rhel_kernel_version_gte 4.18.0-147.el8` in kpatch-build. But I am not sure how to check it here. Is `RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(8, 0)` sufficient?

Reply:
For the .c code, we can use `RHEL_RELEASE_CODE(8, 2)` since atomic replace support was only added in 8.2 GA kernels onward.

For the `kpatch-build` code, we could have used `rhel_kernel_version_gte 4.18.0-147.3.el8` (the trailing `.3` would have been important), but if the kmod code can only check RHEL release numbers, then let's match these two checks.  So then it would be `rhel_kernel_version_gte 4.18.0-193`  as -193 was the 8.2 GA kernel.